src/test/java/com/napier/sem/DAO_IntegrationTests.java

```diff
@@ -279,4 +279,62 @@ void topNCitiesIn_allAreasMatchFilter ()
             // then
             for (City city : cities) assertEquals("United Kingdom", city.getCountry());
         }
+
+        /**
+         * Integration tests covering the DAO.allCapitalCitiesIn method
+         */
+        // Tests that all capital cities in query are in the given area
+        @Test
+        void allCapitalCitiesIn_allAreasMatchFilter()
+        {
+            // given
+            String areaFilter = "Brazil";
+
+            // when
+            ArrayList<CapitalCity> capitalCities = dao.allCapitalCitiesIn(App.COUNTRY, areaFilter);
```

Comment:
this isn't actually functionality that's in the use case for allCapitalCities. The filter should be either continent or region.